packages/cli/src/config/create-server-externals.ts

```diff
@@ -1,10 +1,20 @@
 import fs from 'fs'
-import webpack from 'webpack'
+import { ExternalItemFunctionData } from 'webpack'
 import path from 'path'
 import { CreateWebpackConfigOptions } from '.'
+
+
+type WebpackExternalFunctionElement = ((
+    data: ExternalItemFunctionData,
+    callback: (
+        err?: null | Error,
+        result?: string | boolean | string[] | { [index: string]: any }
+    ) => void
+) => void)
```

Comment:
Instead of redeclaring webpack.ExternalsFunctionElement, you might be able to use `webpack.Externals`:

```typescript
import { Externals } from 'webpack'

export function createServerExternals(options: CreateWebpackConfigOptions): Externals {
    // ...
}
```